libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -682,6 +682,14 @@ class MapboxNavigation(
         fasterRouteController.stop()
     }
 
+    /**
+     * API used to enable/disable route refresh.
+     * By default route refresh is enabled.
+     */
+    fun toggleRouteRefresh(isEnabled: Boolean) {
+        if (isEnabled) routeRefreshController.start() else routeRefreshController.stop()
+    }
```

Comment:
How about we expose this in `NavigationOptions`? Do we expect users to change the refresh state in the runtime?

/cc @asinghal22

Reply:
I don't expect runtime changes.